src/analysis/modals/CromwellModal.js

```diff
@@ -50,7 +52,27 @@ export const CromwellModalBase = withDisplayName('CromwellModal')(
       withErrorReportingInModal('Error creating Cromwell', onError)
     )(async () => {
       if (isAzureWorkspace(workspace)) {
-        await Ajax().Apps.createAppV2(Utils.generateAppName(), workspace.workspace.workspaceId, appToolLabels.CROMWELL);
+        if (isFeaturePreviewEnabled(ENABLE_AZURE_COLLABORATIVE_WORKFLOWS)) {
+          await Ajax().Apps.createAppV2(
+            Utils.generateAppName(),
+            workspace.workspace.workspaceId,
+            appToolLabels.WORKFLOWS_APP,
+            appAccessScopes.WORKSPACE_SHARED
+          );
+          await Ajax().Apps.createAppV2(
+            Utils.generateAppName(),
+            workspace.workspace.workspaceId,
+            appToolLabels.CROMWELL_RUNNER_APP,
+            appAccessScopes.USER_PRIVATE
```

Comment:
Does this need to be passed in the request? My assumption is that Leo will automatically set Cromwell Runner app's access to user private.

Reply:
It doesn't, and I could have just made this the default. It seemed safer to me to make future UI engineers explicitly choose an access scope rather than choosing a default for them